package.json

```diff
@@ -88,6 +88,7 @@
     "exit-code": "^1.0.2",
     "express": "^4.16.4",
     "filesize": "^3.1.3",
+    "form-data": "^2.5.1",
```

Comment:
But...but...we're trying to *reduce* deps?

Reply:
First, typescript the api file. Then, remove request (probably for something else). Adding this dependency (which is already a dependency of `request`, so it's a sum-zero type thing) allows us to have a little better separation :)